Background.md

```diff
@@ -0,0 +1,47 @@
+# SOLID Quest - Achtergrondinformatie
+
+Dit bestand is een verzameling van de kennis en terminologie die we hebben vergaard tijdens onze quest:
+- [Data pods vs data spaces](https://github.com/kadaster-labs/solid-quest/blob/Documentatie/Background.md#data-pods-vs-data-spaces)
```

Comment:
PODs schrijven we overal hetzelfde ... als PODs? Of is Pods en pods ook goed? 🤔

Reply:
Als het goed is staat nu overal POD 🧐